core/app/models/spree/variant.rb

```diff
@@ -83,6 +83,14 @@ class Variant < Spree::Base
       )
     }
 
+    scope :full_in_stock, -> do
+      joins(:stock_items).where(<<~SQL)
```

Comment:
@sebastian-palma this scope is the same as https://github.com/spree/spree/blob/main/core/app/models/spree/variant.rb#L70